ui/app/utils/openapi-to-attrs.js

```diff
@@ -22,6 +22,10 @@ export const expandOpenApiProps = function (props) {
       type = 'number';
     }
 
+    if (prop['x-vault-displayAttrs']?.description) {
```

Comment:
Could we not add `description` above on line 19? Would make this a bit clearer. If we needed to rename it to something so a falsey value doesn't override the previous definition of `description` you could rename it like `{ ..., description: openApiDescription } = prop[...]`

Reply:
Great idea!